src/exceptions/easypost_exception.js

```diff
@@ -0,0 +1,8 @@
+export default class EasyPostException extends Error {
+  constructor({ message, code, statusCode, errors } = {}) {
+    super(message);
+    this.code = code;
+    this.statusCode = statusCode;
+    this.errors = errors;
```

Comment:
An `EasyPostException` just has a message and nothing else. This is because it's the base exception. Library specific errors will not have an array of errors or associated codes since an API call wasn't made. These three parameters need to be on the ApiException object instead.

Remember, we need API errors to extend the `ApiException` and then we need the `ApiException` to extend the `EasyPostException`. General exceptions will also extend the `EasyPostException`. Finally, the `EasyPostException` will extend the builtin Node `Error`.

Reply:
resolved 👍